README_DISCORDBOT.md

```diff
@@ -0,0 +1,18 @@
+# Discord Bot Development
+
+## Getting started
+- Create a bot in discord https://interactions-py.github.io/interactions.py/Guides/02%20Creating%20Your%20Bot/
+- Create your own discord server, add that bot to a test server (feel free to use )
+- Implement bot logics (and add bot token to your code too)
+- Now, run the bot code locally and test the bot on your server
+
+## Auto formatter for Python
```

Comment:
Please leave this for now, until we decided on a unified formatting tool between `ruff`, or use `black` and `flake8`. Possible a @nqngo question?

Reply:
I'm partial to linting tech. However, I am leaning towards the tried and true `black` and `flake8`. Simply because you might be able to slap an upstream `devcontainer.json` and go. I'd imagine you have to do a `apt install` at least on a debian container for `ruff`.

Reply:
nah `ruff` can be installed via poetry, just like `black` and `flake8`. It's quickly becoming the new standard now. 
![image](https://github.com/bifrostlab/llm-assistant/assets/4188758/02f810b8-531d-4f73-8c3f-6fb449417b64)
https://github.com/astral-sh/ruff?tab=readme-ov-file#whos-using-ruff

Perhaps take this out into another discussion?

Reply:
Sounds good. Tested it out in my devcontainer and works wonderfully. Will probably put it as a separated issue. The integration with VSCode is great too.

Reply:
Hey I just put Ruff usage instructions in this PR. Is it ok?